python_backend/api/v1/endpoints/documents.py

```
@@ -22,6 +23,54 @@ def _get_documents_path() -> Path:
+        
+        if not target_dir.is_dir():
+            return []
+            
+        documents = []
+        for root, _, files in os.walk(target_dir):
+            for name in files:
+                # base_dir를 기준으로 한 상대 경로를 반환
+                relative_path = os.path.relpath(os.path.join(root, name), base_dir)
+                documents.append(str(relative_path))
+        
+        return documents
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)